resources/deu.ini

```diff
@@ -1,208 +1,219 @@
+auto.notplotworld=Du bist nicht in einer Grundstückwelt!
+auto.success=Du wurdest erfolgreich zu dem Grundstück ({%0}, {%1}) teleportiert.
+auto.noplots=Es wurde kein verfügbares Grundstück gefunden.
+
 ; The /p dispose Subcommand
 dispose.name=dispose
-dispose.desc=Löscht deinen Plot
+dispose.desc=Entsorge dein Grundstück.
 dispose.usage=/p dispose
-dispose.confirm=Bist du sicher dass du den Plot {%0} löschen möchteset ? Wenn ja, benutze /p dispose confirm
-dispose.nomoney=Du hast nicht genug Geld um den Plot zu löschen!
-dispose.success=Dieser Plot wurde gelöscht! Er gehört dir nun nicht mehr!
+dispose.confirm=Bist du sicher, dass du das Grundstück {%0} entsorgen möchtest? Wenn ja, benutze /p dispose confirm.
+dispose.nomoney=Du besitzt nicht genug Geld, um dieses Grundstück zu entsorgen.
+dispose.success=Das Grundstück wurde entsorgt.
+
 ; The /p reset Subcommand
 reset.name=reset
-reset.desc=Löscht und leert deinen Plot
+reset.desc=Entsorge und leere dein Grundstück
 reset.usage=/p reset
-reset.confirm=Bist du sicher das du den Plot {%0} zurücksetzen möchtest ? Wenn ja, benutze /p reset confirm
-reset.nomoney=Du hast nicht genug Geld um diesen Plot zurückzuzetzen
-reset.success=Plot zurückgesetzt!
+reset.confirm=Bist du sicher, dass du das Grundstück {%0} zurücksetzen möchtest? Wenn ja, benutze /p reset confirm.
+reset.nomoney=Du besitzt nicht genug Geld, um dieses Grundstück zurückzusetzen.
+reset.success=Grundstück zurückgesetzt.
+
 ; The /p clear Subcommand
 clear.name=clear
-clear.desc=Leert deinen Plot
+clear.desc=Leere dein Grundstück
 clear.usage=/p clear
-clear.confirm=Bist du sicher dass du den Plot {%0} leeren möchtest ? Wenn ja, benutze /p clear confirm
-clear.nomoney=Du hast nicht genug Geld um diesen Plot zurückzuzetzen
-clear.success=Plot geleert!
+clear.confirm=Bist du sicher, dass du das Grundstück {%0} leeren möchtest? Wenn ja, benutze /p clear confirm.
+clear.nomoney=Du besitzt nicht genug Geld, um dieses Grundstück zurückzusetzen.
+clear.success=Grundstück geleert.
+
 ; The /p biome Subcommand
 biome.name=biome
-biome.desc=Ändert die Biom von deinem Plot
-biome.usage=/p biome [biome]
-biome.success=Die Plot biome wurde zu  {%0} geändert!
+biome.desc=Ändere das Biom von deinem Grundstück.
+biome.usage=/p biome [Biom]
+biome.success=Das Grundstückbiom wurde zu {%0} geändert.
 biome.invalid=Diese Biom existiert nicht!
-biome.possible=Verfügbare Biomen: {%0}
+biome.possible=Verfügbare Biome: {%0}
 
 ; The /p biomes Subcommand
 biomes.name=biomes
-biomes.desc=Lise von allen verfügbaren Biomen
+biomes.desc=Liste alle verfügbaren Biome auf.
 biomes.usage=/p biomes
-biomes.possible=Verfügbare Biomen sind: {%0}
+biomes.possible=Verfügbare Biome: {%0}
 
 ; The /p name Subcommand
 name.name=name
-name.desc=Setzt oder ändert den Namen von deinem Plot
-name.usage=/p name <name>
-name.success=Der Name deines Plots wurde zu {%0} gesetzt!
+name.desc=Setze den Namen von deinem Grundstück
```

Comment:
"Benenne dein Grundstück um." would sound Germanisher (<-- is that even a word?)